src/generators/go/GoGenerator.ts

```diff
@@ -32,10 +33,14 @@ import { Logger } from '../../utils/LoggingInterface';
 import { GoDefaultConstraints, GoDefaultTypeMapping } from './GoConstrainer';
 import { DeepPartial, mergePartialAndDefault } from '../../utils/Partials';
 import { GoDependencyManager } from './GoDependencyManager';
+import { UnionRenderer } from './renderers/UnionRenderer';
 
 export interface GoOptions extends CommonGeneratorOptions<GoPreset> {
   typeMapping: TypeMapping<GoOptions, GoDependencyManager>;
   constraints: Constraints<GoOptions>;
+  unionAnyModelName: string;
```

Comment:
Please add JS doc to each of these options so people got some idea when trying to use it instead of having to look it up ✌️

Reply:
Will the developer know what `unionAnyModelName` means by reading the description you just provided?